front/src/AppNavbar.tsx

```diff
@@ -16,6 +15,7 @@ import { useIsSubMenuNavbarDisplayed } from '@/ui/layout/hooks/useIsSubMenuNavba
 import MainNavbar from '@/ui/navbar/components/MainNavbar';
 import NavItem from '@/ui/navbar/components/NavItem';
 import NavTitle from '@/ui/navbar/components/NavTitle';
+import { IconBell } from '@tabler/icons-react';
```

Comment:
"import" probably needs to be in alphabetic order.

Reply:
Yes good point @jessicali2023! @ducksblock you can run something like `npx eslint src/ --fix` in the `front` folder to do those fixes automatically

Reply:
@ducksblock Also, please import icons from "@/ui/icon" (see other icons for example)